src/data/HeaderContent.js

```diff
@@ -31,17 +31,13 @@ const Header = () => {
         Benvenuto su Jewelry Shop Olivia Rossi!
       </p>
       <Router>
-        <Route exact path='/login'>
-          <Login />
-        </Route>
+        <Routes>
```

Comment:
The error I was seeing in the browser console:

```
Uncaught Error: [Login] is not a <Route> component. All component children of <Routes> must be a <Route> or <React.Fragment>
```

That means that we need to nest `<Routes>` under `<Router>` to wrap a set of route components.

More info here:

[https://reactrouter.com/en/main/router-components/browser-router](https://reactrouter.com/en/main/router-components/browser-router)

 [https://reactrouter.com/en/main/components/routes](https://reactrouter.com/en/main/components/routes)